Sources/CustomDump/Internal/Box.swift

```diff
@@ -23,6 +23,10 @@ func isMirrorEqual(_ lhs: Any, _ rhs: Any) -> Bool {
     lhsMirror.subjectType == rhsMirror.subjectType,
     lhsMirror.children.count == rhsMirror.children.count
   else { return false }
+  guard !lhsMirror.children.isEmpty, !rhsMirror.children.isEmpty
+  else {
+    return String(describing: lhs) == String(describing: rhs)
```

Comment:
We actually have a bug where `isMirrorEqual(Enum.foo, Enum.bar)` returns `true` where they're the same type but the two cases have no payload.

A few other things we could do to beef up `isMirrorEqual`:

- We could use `memcmp`...not sure it's worth opening that can of worms...
- We could use `enumTag` instead of `String.init(describing:)`, but not sure if `String.init(describing:)` could catch more cases.